operations/security_and_authentication/System_Security_and_Authentication.md

```diff
@@ -35,6 +35,11 @@ All connections through the Istio ingress gateway require authentication with a
     -   [Change EX Liquid-Cooled Cabinet Global Default Password](Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md)
     -   [Provisioning a Liquid-Cooled EX Cabinet CEC with Default Credentials](Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md)
     -   [Updating the Liquid-Cooled EX Cabinet Default Credentials after a CEC Password Change](Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md)
+    -   [Update default Air-Cooled BMC and Leaf switch SNMP credentials](Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md)
```

Comment:
This entire TOC section can be removed. We are only tracking this in /operations/index.md now. It looks like this got added back in a previous PR by mistake.

Reply:
Just to be clear, this entire block can be remove: https://github.com/Cray-HPE/docs-csm/blob/0efffe6f1c88cc3bee73a4196e4369be6e75101a/operations/security_and_authentication/System_Security_and_Authentication.md?plain=1#L29-L76

Reply:
Yes, that is correct.

Reply:
Should be resolved fixed now.